main.cpp

```diff
@@ -56,51 +56,66 @@ Purpose:  This project will show you the difference between member functions and
 
 #include <iostream>
 #include <string>
+
 struct T
 {
-    T(<#type name#> v, const char* <#variable name#>)   //1
+    T(int inValue, const char *inName) : value(inValue), name(inName) {}   //1
     //2
+    int value;
     //3
+    std::string name;
 };
 
-struct <#structName1#>                                //4
+struct Comparator                                //4
 {
-    <#type name#> compare(<#type name#> a, <#type name#> b) //5
+    T *compare(T *a, T *b) //5
```

Comment:
put the `*` next to the type name, not the variable name.
it makes it more clear that it's a pointer.   This also aligns with the JUCE Coding Standard
- [ ] revise